### ISSUE TITLE
`mysqladmin` missing from web container in 1.23.2 (MySQL)

### ISSUE THREAD

**Issue by ajibarra** [ddev/ddev#6355]
### Preliminary checklist

- [X] I am using the latest stable version of DDEV (see [upgrade guide](https://ddev.readthedocs.io/en/stable/users/install/ddev-upgrade/))
- [X] I have searched [existing issues](https://github.com/ddev/ddev/issues)
- [X] I have checked the [troubleshooting guide](https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/)
- [X] I have run `ddev debug test` to include output below

### Output of `ddev debug test`

<details><summary>Expand `ddev debug test` diagnostic information</summary>

```


======== Please make sure that you have already looked at troubleshooting guide ========
Troubleshooting guide: https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/ 
Simple things to check:
* ddev poweroff
* Restart Docker Provider
* Reboot computer
* Temporarily disable VPN and firewall
Press any key to continue:


======== Output file will be in /var/folders/qz/8bss8mpd511d30wmgr3hp4k80000gn/T/ddev-debug-test.txt ========


======== Existing project config ========
ddev installation alternate locations:
/opt/homebrew/bin/ddev

These config files were loaded for project wwis: [/Users/ajibarra/work/Sites/wwis/.ddev/config.yaml]
name: wwis
type: php
docroot: webroot
php_version: 8.2
webserver_type: nginx-fpm
webimage: ddev/ddev-webserver:v1.23.2
router_http_port: 80
router_https_port: 443
xdebug_enabled: true
additional_hostnames: []
additional_fqdns: []
database: {mysql 8.0}
host_db_port: 59001
project_tld: ddev.site
use_dns_when_possible: true
nodejs_version: 16
default_container_timeout: 120


======== existing project customizations ========
.ddev/docker-compose.rabbitmq.yaml
.ddev/php/php-example.ini.example
.ddev/rabbitmq-build/enabled_plugins
.ddev/mysql/my.cnf


======== installed DDEV add-ons ========
No registered add-ons were found. Add-ons installed before DDEV v1.22.0 will not be listed.
Update them with `ddev get` so they'll be shown.


======== mutagen situation ========
looking for #ddev-generated in mutagen.yml in project /Users/ajibarra/work/Sites/wwis
#ddev-generated
unmodified #ddev-generated found in .ddev/mutagen/mutagen.yml


======== Creating dummy project named tryddevproject-13667 in ../tryddevproject-13667 ========
Creating a new DDEV project config in the current directory (/Users/ajibarra/work/Sites/tryddevproject-13667)
Once completed, your configuration will be written to /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/config.yaml

Configuring a 'php' project named 'tryddevproject-13667' with docroot 'web' at '/Users/ajibarra/work/Sites/tryddevproject-13667/web'.
For full details use 'ddev describe'.
Configuration complete. You may now run 'ddev start'.


======== OS Information ========
Darwin libelula.local 23.4.0 Darwin Kernel Version 23.4.0: Fri Mar 15 00:10:42 PDT 2024; root:xnu-10063.101.17~1/RELEASE_ARM64_T6000 arm64
ProductName:		macOS
ProductVersion:		14.4.1
BuildVersion:		23E224


======== User information ========
uid=501(ajibarra) gid=20(staff) groups=20(staff),12(everyone),61(localaccounts),79(_appserverusr),80(admin),81(_appserveradm),98(_lpadmin),33(_appstore),100(_lpoperator),204(_developer),250(_analyticsusers),395(com.apple.access_ftp),398(com.apple.access_screensharing),399(com.apple.access_ssh),400(com.apple.access_remote_ae)


======== DDEV version ========
 ITEM             VALUE                                    
 DDEV version     v1.23.2                                  
 architecture     arm64                                    
 cgo_enabled      0                                        
 db               ddev/ddev-dbserver-mariadb-10.11:v1.23.2 
 ddev-ssh-agent   ddev/ddev-ssh-agent:v1.23.2              
 docker           26.0.0                                   
 docker-api       1.45                                     
 docker-compose   v2.27.0                                  
 docker-platform  docker-desktop                           
 global-ddev-dir  /Users/ajibarra/.ddev                    
 mutagen          0.17.2                                   
 os               darwin                                   
 router           ddev/ddev-traefik-router:v1.23.2         
 web              ddev/ddev-webserver:v1.23.2              



======== proxy settings ========

 HTTP_PROXY=''
 HTTPS_PROXY=''
 http_proxy=''
 NO_PROXY=''
 


======== DDEV global info ========
developer-mode=false
disable-http2=false
fail-on-hook-fail=false
instrumentation-opt-in=true
instrumentation-queue-size=0
instrumentation-reporting-interval=0s
instrumentation-user=
internet-detection-timeout-ms=3000
last-started-version=v1.23.2
letsencrypt-email=
mailpit-http-port=8025
mailpit-https-port=8026
mkcert-caroot=/Users/ajibarra/Library/Application Support/mkcert
no-bind-mounts=false
omit-containers=[]
performance-mode=mutagen
project-tld=ddev.site
required-docker-compose-version=v2.27.0
router=traefik
router-bind-all-interfaces=false
router-http-port=80
router-https-port=443
simple-formatting=false
table-style=default
traefik-monitor-port=10999
use-docker-compose-from-path=false
use-hardened-images=false
use-letsencrypt=false
wsl2-no-windows-hosts-mgt=false
xdebug-ide-location=


======== DOCKER provider info ========
docker client location: lrwxr-xr-x  1 root  wheel  54 Oct 27  2021 /usr/local/bin/docker -> /Applications/Docker.app/Contents/Resources/bin/docker

docker client alternate locations:
/usr/local/bin/docker
/usr/local/bin/docker

Docker provider: docker-desktop
Docker Desktop Version: Docker Desktop for Mac 4.29.0 build 145265
docker version: 
Client:
 Cloud integration: v1.0.35+desktop.13
 Version:           26.0.0
 API version:       1.45
 Go version:        go1.21.8
 Git commit:        2ae903e
 Built:             Wed Mar 20 15:14:46 2024
 OS/Arch:           darwin/arm64
 Context:           default

Server: Docker Desktop 4.29.0 (145265)
 Engine:
  Version:          26.0.0
  API version:      1.45 (minimum version 1.24)
  Go version:       go1.21.8
  Git commit:       8b79278
  Built:            Wed Mar 20 15:18:02 2024
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.28
  GitCommit:        ae07eda36dd25f8a1b98dfbf587313b99c0190bb
 runc:
  Version:          1.1.12
  GitCommit:        v1.1.12-0-g51d5e94
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0

DOCKER_DEFAULT_PLATFORM=notset
Docker platform: docker-desktop
Using Docker context: default (unix:///Users/ajibarra/.docker/run/docker.sock)
docker-compose: v2.27.0
Using DOCKER_HOST=unix:///Users/ajibarra/.docker/run/docker.sock
Docker version: 26.0.0
Able to run simple container that mounts a volume.
Able to use internet inside container.
Docker disk space:Filesystem                Size      Used Available Use% Mounted on
overlay                   1.0T    594.3G    384.5G  61% /


======== Existing docker containers ========
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES


======== mkcert information ========
/opt/homebrew/bin/mkcert
/Users/ajibarra/Library/Application Support/mkcert
total 16
-r--------  1 ajibarra  staff  2484 Oct 28  2021 rootCA-key.pem
-rw-r--r--  1 ajibarra  staff  1777 Oct 28  2021 rootCA.pem


======== ping attempt on ddev.site ========
PING dkdkd.ddev.site (127.0.0.1): 56 data bytes
64 bytes from 127.0.0.1: icmp_seq=0 ttl=64 time=0.023 ms

--- dkdkd.ddev.site ping statistics ---
1 packets transmitted, 1 packets received, 0.0% packet loss
round-trip min/avg/max/stddev = 0.023/0.023/0.023/nan ms


======== ddev debug refresh ========
Rebuilding project images...
Rebuilding web image with `/Users/ajibarra/.ddev/bin/docker-compose -f /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/.ddev-docker-compose-full.yaml build web --no-cache`
#0 building with "default" instance using docker driver

#1 [web internal] load build definition from Dockerfile
#1 transferring dockerfile: 1.25kB done
#1 DONE 0.0s

#2 [web internal] load metadata for docker.io/ddev/ddev-webserver:v1.23.2
#2 DONE 0.0s

#3 [web internal] load .dockerignore
#3 transferring context: 2B done
#3 DONE 0.0s

#4 [web 1/7] FROM docker.io/ddev/ddev-webserver:v1.23.2
#4 CACHED

#5 [web 2/7] RUN (groupadd --gid 20 "ajibarra" || groupadd "ajibarra" || true) && (useradd  -l -m -s "/bin/bash" --gid "ajibarra" --comment '' --uid 501 "ajibarra" || useradd  -l -m -s "/bin/bash" --gid "ajibarra" --comment '' "ajibarra" || useradd  -l -m -s "/bin/bash" --gid "20" --comment '' "ajibarra" || useradd -l -m -s "/bin/bash" --comment '' ajibarra )
#5 0.117 groupadd: GID '20' already exists
#5 0.126 useradd warning: ajibarra's uid 501 outside of the UID_MIN 1000 and UID_MAX 60000 range.
#5 DONE 0.1s

#6 [web 3/7] RUN export XDEBUG_MODE=off; composer self-update --stable || composer self-update --stable || true; composer self-update --2 || composer self-update --2 || true
#6 0.197 Storing "stable" as default update channel for the next self-update run.
#6 0.378 You are already using the latest available Composer version 2.7.7 (stable channel).
#6 0.609 You are already using the latest available Composer version 2.7.7 (2.x channel).
#6 DONE 0.6s

#7 [web 4/7] RUN mkdir -p /home/ajibarra && chown ajibarra /home/ajibarra && chmod 600 /home/ajibarra/.pgpass
#7 DONE 0.1s

#8 [web 5/7] RUN apt-get update
#8 0.278 Get:1 http://deb.debian.org/debian bookworm InRelease [151 kB]
#8 0.338 Get:2 http://deb.debian.org/debian bookworm-updates InRelease [55.4 kB]
#8 0.362 Get:3 http://deb.debian.org/debian-security bookworm-security InRelease [48.0 kB]
#8 0.381 Get:4 https://deb.nodesource.com/node_20.x nodistro InRelease [12.1 kB]
#8 0.385 Get:5 http://deb.debian.org/debian bookworm/main arm64 Packages [8685 kB]
#8 0.482 Get:6 https://deb.nodesource.com/node_20.x nodistro/main arm64 Packages [8094 B]
#8 0.521 Get:7 https://packages.sury.org/php bookworm InRelease [7542 B]
#8 0.564 Get:8 https://packages.sury.org/php bookworm/main arm64 Packages [222 kB]
#8 0.572 Get:9 http://deb.debian.org/debian bookworm-updates/main arm64 Packages [13.7 kB]
#8 0.574 Get:10 http://deb.debian.org/debian-security bookworm-security/main arm64 Packages [161 kB]
#8 0.588 Get:11 http://nginx.org/packages/debian bookworm InRelease [2866 B]
#8 0.648 Get:12 http://nginx.org/packages/debian bookworm/nginx arm64 Packages [12.1 kB]
#8 1.121 Get:14 https://dl.cloudsmith.io/public/platformsh/cli/deb/debian bookworm InRelease [3675 B]
#8 1.337 Get:15 https://dl.cloudsmith.io/public/platformsh/upsun-cli/deb/debian bookworm InRelease [3675 B]
#8 1.498 Get:16 https://dl.cloudsmith.io/public/symfony/stable/deb/debian bookworm InRelease [4422 B]
#8 1.550 Get:13 https://dlm.mariadb.com/repo/mariadb-server/10.11/repo/debian bookworm InRelease [4625 B]
#8 1.681 Get:18 https://dl.cloudsmith.io/public/platformsh/cli/deb/debian bookworm/main arm64 Packages [3299 B]
#8 1.720 Get:17 https://dlm.mariadb.com/repo/maxscale/latest/apt bookworm InRelease [9341 B]
#8 1.800 Get:19 https://dl.cloudsmith.io/public/platformsh/upsun-cli/deb/debian bookworm/main arm64 Packages [2310 B]
#8 1.921 Get:21 https://dl.cloudsmith.io/public/symfony/stable/deb/debian bookworm/main arm64 Packages [7095 B]
#8 2.112 Get:20 https://dlm.mariadb.com/repo/mariadb-server/10.11/repo/debian bookworm/main amd64 Packages [41.5 kB]
#8 2.704 Get:22 https://dlm.mariadb.com/repo/mariadb-server/10.11/repo/debian bookworm/main arm64 Packages [34.9 kB]
#8 2.964 Get:23 https://dlm.mariadb.com/repo/maxscale/latest/apt bookworm/main amd64 Packages [553 B]
#8 3.262 Get:24 https://dlm.mariadb.com/repo/maxscale/latest/apt bookworm/main arm64 Packages [554 B]
#8 3.286 Fetched 9494 kB in 3s (3028 kB/s)
#8 3.286 Reading package lists...
#8 DONE 3.7s

#9 [web 6/7] RUN curl -I https://www.google.com
#9 0.109   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
#9 0.109                                  Dload  Upload   Total   Spent    Left  Speed
#9 0.109 
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
#9 0.286 HTTP/2 200 
#9 0.286 content-type: text/html; charset=ISO-8859-1
#9 0.286 content-security-policy-report-only: object-src 'none';base-uri 'self';script-src 'nonce-WbD-tJeeC1q9l2inYMhWYw' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
#9 0.286 p3p: CP="This is not a P3P policy! See g.co/p3phelp for more info."
#9 0.286 date: Fri, 28 Jun 2024 08:24:28 GMT
#9 0.286 server: gws
#9 0.286 x-xss-protection: 0
#9 0.286 x-frame-options: SAMEORIGIN
#9 0.286 expires: Fri, 28 Jun 2024 08:24:28 GMT
#9 0.286 cache-control: private
#9 0.286 set-cookie: AEC=AQTF6Hyh__G32IeJZ64TgU1zPnDxIiJzZhmrtntu7ZabLmpJmUk0WQgJxA; expires=Wed, 25-Dec-2024 08:24:28 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
#9 0.286 set-cookie: __Secure-ENID=20.SE=cg6F9osi6WiVF83Ti2LweGEoCDjfiRrPsrsBoCaB-uSZ7MEaYro02TfhVracOHXlMUMusrErzcscTOr5hILjgmMphPBGgIejZx6DVbptOoReP4WbAARvpJkPcglfXpM1JMwqNt0hDh_fM9wsLdwte5lCK04JiDhwHtVSZtgCTzlF1lCfuFlopy91AvfamI0arlkX2OLqgxGd; expires=Tue, 29-Jul-2025 00:42:46 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
#9 0.286 alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000
#9 0.286 
#9 DONE 0.3s

#10 [web 7/7] RUN chmod 777 /run/php
#10 DONE 0.1s

#11 [web] exporting to image
#11 exporting layers 0.0s done
#11 writing image sha256:8b3b7ad0fb2278f850d6b50a84685663f40e37468da27e68588fc730930277e6 done
#11 naming to docker.io/ddev/ddev-webserver:v1.23.2-tryddevproject-13667-built done
#11 DONE 0.0s
Refreshed Docker cache for project tryddevproject-13667 in 5s
Network ddev_default created
 Container ddev-ssh-agent  Created
 Container ddev-ssh-agent  Started
ssh-agent container is running: If you want to add authentication to the ssh-agent container, run 'ddev auth ssh' to enable your keys.
v1.23.2: Pulling from ddev/ddev-dbserver-mariadb-10.11
9b10a938e284: Pulling fs layer
331be689e5be: Pulling fs layer
1e6bbb859570: Pulling fs layer
b73d9b054a67: Pulling fs layer
41019807b3c9: Pulling fs layer
e1c9aaf48af4: Pulling fs layer
a1b3ef3d99d5: Pulling fs layer
7271026b1c97: Pulling fs layer
b73d9b054a67: Waiting
4f4fb700ef54: Pulling fs layer
41019807b3c9: Waiting
e1c9aaf48af4: Waiting
a1b3ef3d99d5: Waiting
6c25265908fc: Pulling fs layer
5f125235c07d: Pulling fs layer
7271026b1c97: Waiting
9b122cf0e75e: Pulling fs layer
4f4fb700ef54: Waiting
ebead02e0561: Pulling fs layer
6c25265908fc: Waiting
597384a5c99e: Pulling fs layer
8180c487fd77: Pulling fs layer
1acdf08966b8: Pulling fs layer
b81fdfc679de: Pulling fs layer
f1ddae6d812f: Pulling fs layer
56063cccaa2d: Pulling fs layer
38c417ebaec1: Pulling fs layer
ae3e27c26cda: Pulling fs layer
5f125235c07d: Waiting
d4fb7a7ab524: Pulling fs layer
9b122cf0e75e: Waiting
ebead02e0561: Waiting
597384a5c99e: Waiting
9446ed6677fe: Pulling fs layer
8180c487fd77: Waiting
1acdf08966b8: Waiting
b81fdfc679de: Waiting
f1ddae6d812f: Waiting
56063cccaa2d: Waiting
246f980fabce: Pulling fs layer
38c417ebaec1: Waiting
ae3e27c26cda: Waiting
d4fb7a7ab524: Waiting
9446ed6677fe: Waiting
246f980fabce: Waiting
331be689e5be: Verifying Checksum
331be689e5be: Download complete
1e6bbb859570: Verifying Checksum
1e6bbb859570: Download complete
b73d9b054a67: Download complete
41019807b3c9: Verifying Checksum
41019807b3c9: Download complete
9b10a938e284: Verifying Checksum
9b10a938e284: Download complete
a1b3ef3d99d5: Verifying Checksum
a1b3ef3d99d5: Download complete
7271026b1c97: Verifying Checksum
7271026b1c97: Download complete
4f4fb700ef54: Verifying Checksum
4f4fb700ef54: Download complete
9b10a938e284: Pull complete
331be689e5be: Pull complete
6c25265908fc: Verifying Checksum
6c25265908fc: Download complete
1e6bbb859570: Pull complete
b73d9b054a67: Pull complete
41019807b3c9: Pull complete
9b122cf0e75e: Download complete
ebead02e0561: Verifying Checksum
ebead02e0561: Download complete
597384a5c99e: Verifying Checksum
597384a5c99e: Download complete
5f125235c07d: Verifying Checksum
5f125235c07d: Download complete
8180c487fd77: Verifying Checksum
8180c487fd77: Download complete
1acdf08966b8: Verifying Checksum
1acdf08966b8: Download complete
e1c9aaf48af4: Verifying Checksum
e1c9aaf48af4: Download complete
f1ddae6d812f: Verifying Checksum
f1ddae6d812f: Download complete
b81fdfc679de: Verifying Checksum
b81fdfc679de: Download complete
56063cccaa2d: Verifying Checksum
56063cccaa2d: Download complete
38c417ebaec1: Verifying Checksum
38c417ebaec1: Download complete
ae3e27c26cda: Verifying Checksum
ae3e27c26cda: Download complete
d4fb7a7ab524: Verifying Checksum
d4fb7a7ab524: Download complete
9446ed6677fe: Verifying Checksum
9446ed6677fe: Download complete
246f980fabce: Verifying Checksum
246f980fabce: Download complete
e1c9aaf48af4: Pull complete
a1b3ef3d99d5: Pull complete
7271026b1c97: Pull complete
4f4fb700ef54: Pull complete
6c25265908fc: Pull complete
5f125235c07d: Pull complete
9b122cf0e75e: Pull complete
ebead02e0561: Pull complete
597384a5c99e: Pull complete
8180c487fd77: Pull complete
1acdf08966b8: Pull complete
b81fdfc679de: Pull complete
f1ddae6d812f: Pull complete
56063cccaa2d: Pull complete
38c417ebaec1: Pull complete
ae3e27c26cda: Pull complete
d4fb7a7ab524: Pull complete
9446ed6677fe: Pull complete
246f980fabce: Pull complete
Digest: sha256:6fa68f3698fc312032456c8fd76c354660b183cd5e6e93b06643a917f5355762
Status: Downloaded newer image for ddev/ddev-dbserver-mariadb-10.11:v1.23.2
docker.io/ddev/ddev-dbserver-mariadb-10.11:v1.23.2
Building project images...
.Project images built in 1s.
 Network ddev-tryddevproject-13667_default  Created
 Container ddev-tryddevproject-13667-web  Created
 Container ddev-tryddevproject-13667-db  Created
 Container ddev-tryddevproject-13667-db  Started
 Container ddev-tryddevproject-13667-web  Started
Starting Mutagen sync process...
Mutagen sync flush completed in 1s.
For details on sync status 'ddev mutagen st tryddevproject-13667 -l'
Waiting for containers to become ready: [web db]
Starting ddev-router if necessary...
 Container ddev-router  Created
 Container ddev-router  Started
Waiting for additional project containers to become ready...
All project containers are now ready.


======== Project startup ========
Starting tryddevproject-13667...
2024-06-28T10:25:04.092 Pulled image for ddev/ddev-webserver:v1.23.2
2024-06-28T10:25:04.094 Pulled image for busybox:stable
2024-06-28T10:25:04.098 Pulled image for ddev/ddev-ssh-agent:v1.23.2
2024-06-28T10:25:04.1 Pulled image for ddev/ddev-traefik-router:v1.23.2
2024-06-28T10:25:04.491 GetDockerContext: contextInfo='default unix:///Users/ajibarra/.docker/run/docker.sock'
2024-06-28T10:25:04.491 Using Docker context default (unix:///Users/ajibarra/.docker/run/docker.sock)
2024-06-28T10:25:04.867 chowning /mnt/ddev-global-cache and /var/lib/mysql to 501
2024-06-28T10:25:05.6 done chowning /mnt/ddev-global-cache and /var/lib/mysql to 501
2024-06-28T10:25:05.793 host.docker.internal='' because no other case was discovered
2024-06-28T10:25:06.181 Pulled image for ddev/ddev-dbserver-mariadb-10.11:v1.23.2
2024-06-28T10:25:06.189 Pulled image for ddev/ddev-webserver:v1.23.2
2024-06-28T10:25:06.196 Pulled image for ddev/ddev-ssh-agent:v1.23.2
2024-06-28T10:25:06.203 Pulled image for ddev/ddev-traefik-router:v1.23.2
Building project images...
2024-06-28T10:25:06.216 Executing docker-compose -f /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/.ddev-docker-compose-full.yaml build --progress=plain
.Project images built in 1s.
2024-06-28T10:25:07.224 Executing docker-compose -f /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/.ddev-docker-compose-full.yaml up -d
 Container ddev-tryddevproject-13667-db  Recreate
 Container ddev-tryddevproject-13667-web  Recreate
 Container ddev-tryddevproject-13667-db  Recreated
 Container ddev-tryddevproject-13667-web  Recreated
 Container ddev-tryddevproject-13667-db  Started
 Container ddev-tryddevproject-13667-web  Started
2024-06-28T10:25:18.135 VIRTUAL_HOST=tryddevproject-13667.ddev.site for web
2024-06-28T10:25:18.135 HTTP_EXPOSE=80:80,8025:8025 for web
2024-06-28T10:25:18.135 HTTPS_EXPOSE=443:80,8026:8025 for web
2024-06-28T10:25:18.563 Copied /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/traefik:CopyIntoVolume_jgtnmjgdowir into /mnt/v/traefik in 65.790875ms
2024-06-28T10:25:18.605 Exec chown -R 501 /mnt/v/traefik stdout=, stderr=, err=<nil>
2024-06-28T10:25:18.758 Copied Traefik certs in /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/traefik/certs to ddev-global-cache/traefik
Starting Mutagen sync process...
2024-06-28T10:25:18.865 Using Mutagen config file /Users/ajibarra/work/Sites/tryddevproject-13667/.ddev/mutagen/mutagen.yml
2024-06-28T10:25:20.078 Mutagen status after sync: ok
Mutagen sync flush completed in 1s.
For details on sync status 'ddev mutagen st tryddevproject-13667 -l'
2024-06-28T10:25:20.079 Running /start.sh in ddev-webserver
Waiting for containers to become ready: [web db]
2024-06-28T10:25:26.026 Copied /Users/ajibarra/.ddev/commands:CopyIntoVolume_gxjexxfksmzx into /mnt/v/global-commands in 62.743917ms
2024-06-28T10:25:26.051 Exec chown -R 501 /mnt/v/global-commands stdout=, stderr=, err=<nil>
2024-06-28T10:25:26.208 Testing to see if /mnt/ddev_config is properly mounted
Starting ddev-router if necessary...
2024-06-28T10:25:26.65 Copied /Users/ajibarra/.ddev/traefik:CopyIntoVolume_xdfafagjvuwy into /mnt/v/traefik in 48.556042ms
2024-06-28T10:25:26.68 Exec chown -R 501 /mnt/v/traefik stdout=, stderr=, err=<nil>
2024-06-28T10:25:26.77 Copied global Traefik config in /Users/ajibarra/.ddev/traefik/certs to ddev-global-cache/traefik
 Container ddev-router  Running
2024-06-28T10:25:26.836 Waiting for ddev-router to become ready. docker inspect --format "{{json .State.Health }}" ddev-router
2024-06-28T10:25:27.34 ddev-router is ready
Waiting for additional project containers to become ready...
All project containers are now ready.
Successfully started tryddevproject-13667
Project can be reached at https://tryddevproject-13667.ddev.site https://127.0.0.1:54712


======== Curl of site from inside container ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0HTTP/1.1 200 OK
Server: nginx
Date: Fri, 28 Jun 2024 08:25:29 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Vary: Accept-Encoding


  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0


======== curl -I of http://127.0.0.1:54713 (web container http docker bind port) from outside ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 28 Jun 2024 08:25:29 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Vary: Accept-Encoding



======== curl -I of http://tryddevproject-13667.ddev.site (router http URL) from outside ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
HTTP/1.1 200 OK
Content-Type: text/html; charset=UTF-8
Date: Fri, 28 Jun 2024 08:25:29 GMT
Server: nginx
Vary: Accept-Encoding



======== Full curl of http://tryddevproject-13667.ddev.site (router http URL) from outside ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   286    0   286    0     0  78787      0 --:--:-- --:--:-- --:--:-- 95333
Success accessing database... db via TCP/IP<br />
ddev is working.<br />
The output file for Discord or issue queue is in
<b>/var/folders/qz/8bss8mpd511d30wmgr3hp4k80000gn/T/ddev-debug-test.txt</b><br />
file:///var/folders/qz/8bss8mpd511d30wmgr3hp4k80000gn/T/ddev-debug-test.txt<br />


======== Full curl of https://tryddevproject-13667.ddev.site (router https URL) from outside ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100   286    0   286    0     0   5047      0 --:--:-- --:--:-- --:--:--  5107
Success accessing database... db via TCP/IP<br />
ddev is working.<br />
The output file for Discord or issue queue is in
<b>/var/folders/qz/8bss8mpd511d30wmgr3hp4k80000gn/T/ddev-debug-test.txt</b><br />
file:///var/folders/qz/8bss8mpd511d30wmgr3hp4k80000gn/T/ddev-debug-test.txt<br />


======== Curl google.com to check internet access and VPN ========
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
HTTP/2 200 
content-type: text/html; charset=ISO-8859-1
content-security-policy-report-only: object-src 'none';base-uri 'self';script-src 'nonce-TuekaiqjtRwMu1CA_odEKw' 'strict-dynamic' 'report-sample' 'unsafe-eval' 'unsafe-inline' https: http:;report-uri https://csp.withgoogle.com/csp/gws/other-hp
p3p: CP="This is not a P3P policy! See g.co/p3phelp for more info."
date: Fri, 28 Jun 2024 08:25:29 GMT
server: gws
x-xss-protection: 0
x-frame-options: SAMEORIGIN
expires: Fri, 28 Jun 2024 08:25:29 GMT
cache-control: private
set-cookie: AEC=AQTF6Hx3Rw2Aq1I6MjXdQL5FCjVEmitv6nTeBSSbJDX_Z3x8RHU7_NnDuKg; expires=Wed, 25-Dec-2024 08:25:29 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
set-cookie: __Secure-ENID=20.SE=b14Fa3A9mBXNJHMeanYpahTS6eBbb1-0jalpFue53V8KbMD9_WDJa3TpPGdVNtbzYGl1_XgDJeIoHOr5gHFDeW7MZyvn5xAGbY8OM48QsBonl7WeMtrQo29UQ7rCQnL_9L1NZ60wabxp0AemOFtTl2QVwijsTKRMF18b_oOwpsPligxQWrB8iLelspqouLLuScFfrWeDyZ9Iiw; expires=Tue, 29-Jul-2025 00:43:47 GMT; path=/; domain=.google.com; Secure; HttpOnly; SameSite=lax
alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000



======== host.docker.internal status ========
PING host.docker.internal (192.168.65.254) 56(84) bytes of data.
64 bytes from 192.168.65.254 (192.168.65.254): icmp_seq=1 ttl=63 time=0.669 ms

--- host.docker.internal ping statistics ---
1 packets transmitted, 1 received, 0% packet loss, time 0ms
rtt min/avg/max/mdev = 0.669/0.669/0.669/0.000 ms


======== Project ownership on host ========
drwxr-xr-x  4 ajibarra  staff  128 Jun 28 10:24 ../tryddevproject-13667


======== Project ownership in container ========
drwxr-xr-x 4 ajibarra dialout 4096 Jun 28 08:24 //var/www/html


======== In-container filesystem ========
Filesystem     Type  1K-blocks      Used Available Use% Mounted on
/dev/vda1      ext4 1081328468 623986368 402380412  61% /var/www


======== Thanks for running the diagnostic! ========
Running ddev launch in 3 seconds
Running ddev launch
Waiting for ddev launch to complete before deleting project


Cleanup: deleting test project tryddevproject-13667
 Container ddev-tryddevproject-13667-db  Stopped
```
</details>

### Expected Behavior

I would expect `mysqladmin` to be available in web container

### Actual Behavior

In 1.23.0 `mysqladmin` command was available but it is not in latest version.

### Steps To Reproduce

1. Create a new project with Mysql 8.0 as database
2. `ddev ssh`
3. Try to execute `mysqladmin`
4. You will get a commant not found error

**Comment by stasadev** [comment]
It has been removed here, now only `mysql` and `mysqldump` are available in the webserver container:

- #6297

@rfay, I think we should reconsider adding more `mysql*` commands.

**Comment by rfay** [comment]
This was also discussed in Discord: https://discord.com/channels/664580571770388500/1255231270343151698/1255231270343151698

Please note that mysqladmin *is* available, with the correct and matching version, in the `db` container. `ddev ssh -s db` and use `mysqladmin -uroot -proot`

**Comment by rfay** [comment]
@stasadev I agree, we're already building the additional clients anyway.

**Comment by rfay** [comment]
AFAICT this affects only MySQL users (mysql:5.7 and mysql:8.0) running on ARM64 systems. Users of MariaDB (I think all versions) have these clients.

**Comment by rfay** [comment]
@ajibarra could you please test the artifacts in 
* https://github.com/ddev/ddev/pull/6360#issuecomment-2198246650

and see if they meet your needs? Thanks!

**Comment by ajibarra** [comment]
> @ajibarra could you please test the artifacts in
> 
> * [fix: add mysqladmin to web container for mysql, fixes #6355 #6360 (comment)](https://github.com/ddev/ddev/pull/6360#issuecomment-2198246650)
> 
> and see if they meet your needs? Thanks!

Tested. Everything working great! 

Thank you @rfay!

**Comment by rfay** [comment]
Thanks!
